ui/admin/tests/acceptance/credential-store/credentials/create-test.js

```diff
@@ -0,0 +1,124 @@
+import { module, test } from 'qunit';
+import { visit, currentURL, find, click, fillIn } from '@ember/test-helpers';
+import { setupApplicationTest } from 'ember-qunit';
+import setupMirage from 'ember-cli-mirage/test-support/setup-mirage';
+import { authenticateSession } from 'ember-simple-auth/test-support';
+import { Response } from 'miragejs';
+
+module(
+  'Acceptance | credential-stores | credentials | create',
+  function (hooks) {
+    setupApplicationTest(hooks);
+    setupMirage(hooks);
+
+    let getCredentialsCount;
+
+    const instances = {
+      scopes: {
+        org: null,
+        project: null,
+      },
+    };
+
+    const urls = {
+      projectScope: null,
+      credentialStores: null,
+      staticCredentialStore: null,
+      credentials: null,
+      newCredential: null,
+    };
+
+    hooks.beforeEach(function () {
+      // Generate resources
+      instances.scopes.org = this.server.create('scope', {
+        type: 'org',
+        scope: { id: 'global', type: 'global' },
+      });
+      instances.scopes.project = this.server.create('scope', {
+        type: 'project',
+        scope: { id: instances.scopes.org.id, type: 'org' },
+      });
+      instances.staticCredentialStore = this.server.create('credential-store', {
+        scope: instances.scopes.project,
+        type: 'static',
+      });
+      // Generate route URLs for resources
+      urls.projectScope = `/scopes/${instances.scopes.project.id}`;
+      urls.credentialStores = `${urls.projectScope}/credential-stores`;
+      urls.staticCredentialStore = `${urls.credentialStores}/${instances.staticCredentialStore.id}`;
+      urls.credentials = `${urls.staticCredentialStore}/credentials`;
+      urls.newCredential = `${urls.staticCredentialStore}/credentials/new`;
+      // Generate resource counter
+      getCredentialsCount = () => {
```

Comment:
There is a reason this function is re-declared here?

Because this is a `beforeEach` block, we are re-decalring this functions for every test. Can we declare this function out of the `beforeEach`?

Reply:
I am mainly following conventions from previous tests. However, upon further inspection this.server.schema is not accessible (undefined) outside `beforeEach` and `test` blocks.

Reply:
@lisbet-alvarez fair enough. You are right, best follow previous patterns. Will work later on on improvimg them if there is time 😉